addons/dark-www/addon.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Dark WWW",
+  "description": "Dark Theme for Scratch Website",
```

Comment:
Descriptions should use sentence case and end with a dot, see #104.

Reply:
ok, ok!

Reply:
added a dot